cmd/create/cluster/cmd.go

```diff
@@ -3017,6 +3090,34 @@ func run(cmd *cobra.Command, _ []string) {
 	}
 }
 
+func PrintContracts(contracts []*accountsv1.Contract, r *rosa.Runtime) {
+	for _, contract := range contracts {
+		format := "Jan 02, 2006"
+		dimensions := contract.Dimensions()
+		var numberOfString string
+
+		for _, dimension := range dimensions {
+			numberOfVPCs := ocm.GetDimensionValue(dimension, "four_vcpu_hour")
+			numberOfClusters := ocm.GetDimensionValue(dimension, "control_plane")
+			numberOfString = numberOfString + fmt.Sprintf("   | Number of vCPUs:    |'%s'             | \n"+
```

Comment:
nit: `numberOfString` is not mnemonic - it might be worth it to find another name.

Reply:
How about prePurchaseInfo?

Reply:
Updated with prePurchaseInfo, but we can change the name